src/styles/typography.ts

```diff
@@ -39,57 +41,107 @@ export const fontWeight: Record<FontWeight, TextStyle> = {
 type LineHeight = "x10" | "x20" | "x30" | "x40" | "x50" | "x60"
 export const lineHeight: Record<LineHeight, TextStyle> = {
   x10: {
-    lineHeight: 12,
+    lineHeight: 18,
   },
   x20: {
-    lineHeight: 18,
+    lineHeight: 22,
   },
   x30: {
-    lineHeight: 22,
+    lineHeight: 26,
   },
   x40: {
-    lineHeight: 26,
+    lineHeight: 32,
   },
   x50: {
-    lineHeight: 36,
+    lineHeight: 38,
   },
   x60: {
-    lineHeight: 48,
+    lineHeight: 44,
   },
 }
 
-type Header = "x10" | "x20" | "x40" | "x50" | "x60"
+type Header = "x10" | "x20" | "x30" | "x40" | "x50" | "x60"
 export const header: Record<Header, TextStyle> = {
   x10: {
     ...fontSize.x10,
-    ...lineHeight.x20,
     ...fontWeight.bold,
+    ...lineHeight.x10,
   },
   x20: {
     ...fontSize.x20,
+    ...fontWeight.bold,
+    ...lineHeight.x20,
+  },
+  x30: {
+    ...fontSize.x30,
+    ...fontWeight.bold,
     ...lineHeight.x30,
-    ...fontWeight.semibold,
   },
   x40: {
     ...fontSize.x40,
+    ...fontWeight.bold,
     ...lineHeight.x40,
-    ...fontWeight.semibold,
   },
   x50: {
     ...fontSize.x50,
-    ...lineHeight.x50,
     ...fontWeight.bold,
+    ...lineHeight.x50,
   },
   x60: {
     ...fontSize.x60,
-    ...lineHeight.x60,
     ...fontWeight.bold,
+    ...lineHeight.x60,
   },
 }
 
-type Body = "x10"
+type Subheader = "x10" | "x20" | "x30" | "x40"
+export const subheader: Record<Subheader, TextStyle> = {
+  x10: {
+    ...fontSize.x10,
+    ...fontWeight.semibold,
+    ...lineHeight.x10,
+  },
+  x20: {
+    ...fontSize.x20,
+    ...fontWeight.semibold,
+    ...lineHeight.x20,
+  },
+  x30: {
+    ...fontSize.x30,
+    ...fontWeight.semibold,
+    ...lineHeight.x30,
+  },
+  x40: {
+    ...fontSize.x40,
+    ...fontWeight.semibold,
+    ...lineHeight.x40,
+  },
+}
+
+type Body = "x10" | "x20" | "x30" | "x40"
 export const body: Record<Body, TextStyle> = {
   x10: {
     ...fontSize.x10,
+    ...fontWeight.regular,
```

Comment:
It looks like the only difference between body and header is the fontweight? I find this surprising as I would not expect smallest body to be the same size as the smallest header.  While I like the fact that theres consistency in what 'x10' means in this case (it means fontSize.x10), I would imagine that while building components, I would want them to be on different scales.  so body.x30 would be about the same text height as header.x10.

It's probably something I would get used to either way, so not sure it matters too much, but was wondering your thoughts on why what makes the most sense for the scales between the different typography groups.